infrastructure/src/main/java/org/corfudb/infrastructure/SequencerServer.java

```diff
@@ -485,26 +495,40 @@ private void handleAllocation(CorfuPayloadMsg<TokenRequest> msg,
 
         // update the cache of conflict parameters
         if (req.getTxnResolution() != null) {
+            req.getTxnResolution().getPoisonedStreams()
+                    .stream()
+                    .forEach(id -> {
+                        SequencerStreamInfo streamInfo =
+                                conflictToGlobalTailCache
+                                        .computeIfAbsent(id,
+                                                k -> new SequencerStreamInfo());
+                        streamInfo.minPoisoned = newTail - 1;
```

Comment:
how about clearing the conflict map here to reclaim the space? this way, when a map is cleared, the sequencer space is compacted. otherwise, it grows forever.